pydatastructs/miscellaneous_data_structures/stack.py

```diff
@@ -84,6 +84,8 @@ def peek(self):
 
 class ArrayStack(Stack):
 
+    __slots__ = ['maxsize', 'top', 'items']
```

Comment:
Add `dtype` to the list as well.

Reply:
Don't make this change.
I will give you a `diff` by tonight, which you can apply on top of your changes and push them here.

Reply:
ok. I will wait for your changes.